### ISSUE TITLE
OBS builds not getting triggered

### ISSUE THREAD

**Issue by tamara-schmitz** [MrGlockenspiel/activate-linux#80]
The home repository of @WoMspace on OBS does not properly trigger rebuilds.

Please either get @WoMspace to trigger a rebuild manually, set up webhooks properly or create a new home repository.

**Comment by Ruby-Dragon** [comment]
Well turns out we dont have obs anymore :(

**Comment by tamara-schmitz** [comment]
> Well turns out we dont have obs anymore :(

You can use https://build.opensuse.org to build openSUSE and Fedora packages. It can also build packages for other distros like Debians and Arch, but not sure how much it takes to set that up

**Comment by MrGlockenspiel** [comment]
We talked to the maintainer of the OBS repo for it and in the end they decided to drop it
If someone else wants to start a new one, feel free to let me know and I'll set up the webhooks and stuff on my end

**Comment by tamara-schmitz** [comment]
> If someone else wants to start a new one, feel free to let me know and I'll set up the webhooks and stuff on my end

I started one essentially. It currently builds from my fork, but once my PR is merged, I can switch back to this repo. https://github.com/MrGlockenspiel/activate-linux/pull/118

Or if you are uncomfortable with sharing tokens, I can walk you through setting this up yourself.
Currently the package that I set up builds for both Fedora and openSUSE variations: https://build.opensuse.org/package/show/home:tschmitz:activate-linux/activate-linux
